catalog/catalog.html

```diff
@@ -220,71 +251,83 @@ <h2 class="catalog__title title">Каталог продукции</h2>
               На самом деле вкусов гораздо больше!
             </p>
             <button class="catalog__btn--disabled button" type="button">
-              Заказать
+              Показать все
             </button>
           </li>
         </ul>
       </section>
 
       <section class="more-goods">
+        <div class="more-goods__line"></div>
         <div class="more-goods__wrapper container">
-          <table class="more-goods__table">
-            <tr class="more-goods__row">
-              <td class="more-goods__cell more-goods__name">Сахарозаменить</td>
-              <td class="more-goods__cell">1 Упаковка (100 г)</td>
-              <td class="more-goods__cell">200 р.</td>
-              <td class="more-goods__cell">
-                <button class="more-goods__btn button">Заказать</button>
-              </td>
-            </tr>
-            <tr class="more-goods__row">
-              <td class="more-goods__cell more-goods__name">Питьевая вода</td>
-              <td class="more-goods__cell">5 литров</td>
-              <td class="more-goods__cell">50 р.</td>
-              <td class="more-goods__cell">
-                <button class="more-goods__btn button">Заказать</button>
-              </td>
-            </tr>
-            <tr class="more-goods__row">
-              <td class="more-goods__cell more-goods__name">Молоко</td>
-              <td class="more-goods__cell">1 литр</td>
-              <td class="more-goods__cell">100 р.</td>
-              <td class="more-goods__cell">
-                <button class="more-goods__btn button">Заказать</button>
-              </td>
-            </tr>
-            <tr class="more-goods__row">
-              <td class="more-goods__cell more-goods__name">Витамины</td>
-              <td class="more-goods__cell">1 упаковка (30 г)</td>
-              <td class="more-goods__cell">300 р.</td>
-              <td class="more-goods__cell">
-                <button class="more-goods__btn button">Заказать</button>
-              </td>
-            </tr>
-          </table>
-          <div class="more-goods__gift">
-            <img
-              class="more-goods__gift-pic"
-              src="../img/gift.svg"
-              width="55"
-              height="65"
-              alt="Подарок"
-            />
-            <p class="more-goods__offer">
-              Закажите все и получите чехол для кота в подарок!
-            </p>
+          <h3 class="more-goods__title">Дополнительные товары</h3>
+
+          <div class="more-goods__flex-cont">
+            <table class="more-goods__table" cellspacing="0">
+              <tr class="more-goods__row">
+                <td class="more-goods__name">Сахарозаменить</td>
```

Comment:
I think we can semantically treat this as a heading, so `th` would work here better.

Reply:
fixed